AmazonChimeSDKDemo/AmazonChimeSDKDemoObjC/ViewControllerObjC.m

```diff
@@ -360,4 +369,16 @@ - (void)videoTileSizeDidChangeWithTileState:(VideoTileState *)tileState {
     [self.logger infoWithMsg:[NSString stringWithFormat:@"Video Tile size changed: tileId: %ld, attendeeId: %@", (long)tileState.tileId, tileState.attendeeId]];
 }
 
+# pragma mark - ActiveSpeakerObserver
+
+- (void)activeSpeakerScoreDidChangeWithScores:(NSDictionary<AttendeeInfo *, NSNumber *> * _Nonnull)scores {
+    [self.logger infoWithMsg:@"activeSpeakerScoreDidChangeWithScores callback invoked"];
```

Comment:
Have you checked if this callback was invoked? I think it won't be invoked unless you specified `scoresCallbackIntervalMs`. Ref: https://github.com/aws/amazon-chime-sdk-ios/blob/master/AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerdetector/ActiveSpeakerObserver.swift#L20

Reply:
You are right. Implemented scoresCallbackIntervalMs and verified it's getting called